libs/pages/elewa/home/src/lib/pages/home-section/home-section.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="home-section">
+
+    <div class="main-parent">
+
+        <div class="secondary-parent">
+            <h1>Become part of the family</h1>
+            <h1 class="h1-down">View <a href="#">open opportunities.</a></h1>
+            
+        </div>
+
+    </div>
+  
+</div>
```

Comment:
You're renamed and removed the home-page component, this is not acceptable.

Your assignment was to add a new component in the components folder with your updates. Sync and check folder structure